### ISSUE TITLE
Events consumed by default

### ISSUE THREAD

**Issue by arvind** [vega/vega-view#12]
`view._preventDefault` is `true` by default. As far as I can tell, this results in the `consume` flag on event streams having no effect whatsoever. For instance, with the following spec, I would only expect `wheel` events to not scroll the page if they occur over the black rectangle. However, regardless of the status of the `consume` property (`true` or `false`), no scrolling occurs when `wheel` events occur within the entire view.

If `view.preventDefault(false)` is set, then the `consume` flag operates as I would expect. 

Should `view._preventDefault` be `false` by default? Regardless, we should document this method as part of the view API (and reference when discussing event stream consumption). 

```json
{
  "$schema": "https://vega.github.io/schema/vega/v3.0.json",
  "width": 200,
  "height": 200,
  "encode": {"update": {"fill": {"value": "transparent"}}},
  "signals": [
    {
      "name": "brush_zoom_anchor",
      "on": [
        {
          "events": [
            {
              "source": "scope",
              "type": "wheel",
              "marktype": "rect",
              "consume": true
            }
          ],
          "update": "warn({x: x(), y: y()})"
        }
      ]
    }
  ],
  "marks": [
    {
      "type": "rect",
      "encode": {
        "enter": {
          "x": {"value": 50},
          "y": {"value": 0},
          "width": {"value": 50},
          "height": {"value": 50},
          "fill": {"value": "black"}
        }
      }
    }
  ]
}
```

**Comment by jheer** [comment]
We prevent default intentionally just for events originating from the View canvas/svg. First, this is what Vega 2 did. Second, if we don't do this we can get a number of annoying artifacts (e.g., unwanted text selection in response to mouse drag in SVG). It seems that the issue here is with the scroll `wheel` event, which typically has a _global_ interpretation even when triggered upon a _local_ element.

Perhaps it would suffice to omit `wheel` events from consideration here? Are there related events with global interpretations that we should be concerned about? Perhaps with touch input?

**Comment by jheer** [comment]
Addressed in vega-view v1.2.0+ and related vega-parser updates. Specifications now support a `config.events` block that can include a `defaults` property indicating which events to prevent or allow. See forthcoming Vega config documentation for more.

The new default setting is to consume all view-originating events except for `wheel`.
